### PR TITLE
cleanup transitioningIn/transitioningOut

### PR DESCRIPTION
**Pull request by bekzod**

### CODE REVIEW

**Review by rwjblue — APPROVED** [review]

File: packages/ember-glimmer/lib/components/link-to.js [hunk 1]
```diff
@@ -594,17 +594,19 @@ const LinkComponent = EmberComponent.extend({
   }),
 
   transitioningIn: computed('active', 'willBeActive', function computeLinkToComponentTransitioningIn() {
-    let willBeActive = get(this, 'willBeActive');
-    if (typeof willBeActive === 'undefined') { return false; }
-
-    return !get(this, 'active') && willBeActive && 'ember-transitioning-in';
+    if (get(this, 'willBeActive') === true && !get(this, 'active')) {
```
Comment (locks) [inline packages/ember-glimmer/lib/components/link-to.js]: I'm wondering why the mix of comparing to a boolean on the first clause, but using `!` on the second one?

Comment (bekzod) [inline packages/ember-glimmer/lib/components/link-to.js]: they might return
`get(this, 'active')` = > false, undefined, string
`get(this, 'willBeActive')` => true, false, undenfined (I am removing this case)

Comment (bekzod) [inline packages/ember-glimmer/lib/components/link-to.js]: I think I can check for explicit `false`, let me test

Comment (bekzod) [inline packages/ember-glimmer/lib/components/link-to.js]: didn't work `!get(this, 'active')` is correct way for now

**Pull request merged**